integration-cli/docker_cli_events_test.go

```diff
@@ -628,7 +628,7 @@ func (s *DockerSuite) TestEventsCopy(c *check.C) {
 		c.Fatalf("Failed getting file:%q\n%q", out, err)
 	}
 
-	cmd = exec.Command(dockerBinary, "events", "-f", "container=cptest", "--until="+strconv.Itoa(int(since)))
+	cmd = exec.Command(dockerBinary, "events", "--since=0", "-f", "container=cptest", "--until="+strconv.Itoa(int(since)))
```

Comment:
Here's what the docs currently say:

>   --since=""         Show all events created since timestamp

And there are several examples of what timestamps can look like.
I don't think there's any need for a docs change here.